src/DropdownFilter.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.DropdownFilter.DropdownFilter" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
```

Comment:
id should all be lower cases except for the class name (last `DropdownFilter`) =>
`com.mendix.widget.custom.dropdownfilter.DropdownFilter`

Reply:
Done